azimuth/app.py

```diff
@@ -288,3 +282,24 @@ def run_validation_module(pipeline_index=None):
             run_validation_module(pipeline_index)
     task_manager.clear_worker_cache()
     task_manager.restart()
+
+
+def run_start_up_tasks(azimuth_config: AzimuthConfig, cluster: SpecCluster):
```

Comment:
I think we use `startup` everywhere else. Technically, it should be "start-up", but we can't use an hyphen here.
```suggestion
def run_startup_tasks(azimuth_config: AzimuthConfig, cluster: SpecCluster):
```